pandas/tests/extension/test_datetime.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+import pytest
+
```

Comment:
Is if feasible to implement the analogous tests for TimedeltaArray?

Reply:
No. TimedeltaArray.dtype isn't an extension dtype.

Reply:
OK.  We'll need to make sure in follow-up(s) that we have decent coverage for TDA methods